supports/integration-test/test_S3_1MB.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import os
+from util import *
+
+
+FILE_SIZE = 1024 * 1024
+DEST_DIR = "s3a://xxxctest/"
+
+#To run this script, we must install hadoop and enable S3 support in hadoop
+
+
+class Test_S3_1MB(unittest.TestCase):
+
+    # move to S3
+    def test_S3(self):
+        max_number = 1000
+        file_paths = []
+        cids = []
+
+        #create random directory
+        source_dir  = "/" + random_string() + "/"
+        #create 1K random files in random directory
+        for i in range(max_number):
+            file_paths.append(
+                create_random_file_parallel_return_file_name(source_dir, FILE_SIZE))
```

Comment:
Content in file_paths is not correct, i.e., `file_name, submit_cmdlet(cmdlet_str)`.